src/sidebar/components/top-bar.js

```diff
@@ -52,16 +69,21 @@ function TopBar({
     store => store.getState().sidebarPanels.activePanelName
   );
 
+  const searchQueryProp = {};
```

Comment:
This is just a trick to not to pass the prop rather than passing an empty string which is a lot less ugly, however,  not  as correct.